aries-site/src/components/cards/ContentCard.js

```diff
@@ -21,8 +21,29 @@ export const ContentCard = forwardRef(({ topic, ...rest }, ref) => {
       {...rest}
     >
       <Box gap="large">
+        {previewComponent && (
+          <Box
+            background={previewComponent.cardColor || 'background-contrast'}
+            height="small"
+            round="xsmall"
+            overflow="hidden"
+            style={{ position: 'relative' }}
+          >
```

Comment:
It seems like this is duplicative of lines 45-51 and should be combined. Then the conditional should move inside this box and present either the previewComponent or the previewImage.